### PR TITLE
VIVI-1837 YouTube Playlist Support

### PR DESCRIPTION
**Pull request by deanylev**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,7 +3,18 @@ const EN_LIST = ['en-US', 'en-GB', 'en', 'en-AU'];
 
 // public
 
-module.exports.ARGUMENTS = ['--restrict-filenames', '--write-sub', '--write-auto-sub', '--max-downloads', '1', '-f', '[height <=? 720][format_id != source]', '-j'];
+module.exports.isPlaylist = (url) => {
+  return url.startsWith('https://www.youtube.com/playlist?list=');
+};
+
+module.exports.getArguments = (isPlaylist) => {
```
Comment (DuBistKomisch) [inline index.js]: breaking API for no reason? why not just have a separate property with the playlist args

Comment (deanylev) [reply index.js]: Oh tru

File: index.js [hunk 2]
```diff
@@ -26,6 +33,13 @@ module.exports.process = (output, origin) => {
   };
 };
 
+module.exports.processPlaylist = (output) => {
+  return output.toString().split('\n').map((string) => {
```
Comment (DuBistKomisch) [inline index.js]: should you `trim` before `split`ting to avoid getting an extra empty line?

**Review by DuBistKomisch — APPROVED** [review]
seems fine

**Pull request merged**